### PR TITLE
chore: update ImageEmptyPage to include pullImage button

### LINKED ISSUE #8569: Default actions for Images empty state
There should be two options in the following screen to allow the user to quickly get started:
Screenshot 2024-08-25 at 13 05 00

The first one should be a button:
"Pull `podman/hello` Image " 

Then in the screen, we display a label "OR" and the line bellow is the actual commande line we already have in the screen

### PR DESCRIPTION
**Pull request by SoniaSandler**
### What does this PR do?
This PR adds the option to pull a first image in `ImageEmptyPage` with a click of a button.
(Follows the design of  https://github.com/containers/podman-desktop/pull/8651)

### Screenshot / video of UI

[Screencast from 2024-09-15 19-59-51.webm](https://github.com/user-attachments/assets/ecfec495-7857-4908-b39c-6734d8af2e5a)

### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/8569

### How to test this PR?
Access the `ImageEmptyPage`, click the pull image button and confirm that `quay.io/podman/hello` is added to the images list

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/image/ImageEmptyScreen.svelte [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+      title: `Error while pulling image`,
+      message: `No provider connections found`,
+    });
+    return;
+  }
+
+  pullInProgress = true;
+  try {
+    await window.pullImage(selectedProviderConnection, firstImageName, () => {});
+    pullInProgress = false;
```
Comment (deboer-tim) [inline packages/renderer/src/lib/image/ImageEmptyScreen.svelte]: Redundant with line 37, can remove this one.

**Review by deboer-tim — APPROVED** [review]
Just remove redundant line.

**Review by benoitf — DISMISSED** [review]
After the click, for some time, the action is done (no more in progress but I still see the widget)

I don't know if we need to wait the inProgress until images are updated (check that images list is >0 before stopping the inProgress)


Request change as there is no unit tests.

**Review by benoitf — APPROVED** [review]

File: packages/renderer/src/lib/image/ImageEmptyScreen.spec.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
```
Comment (benoitf) [inline packages/renderer/src/lib/image/ImageEmptyScreen.spec.ts]:
```suggestion
 * Copyright (C) 2024 Red Hat, Inc.
```

**Pull request merged**